docs/install/sandboxes.rst

```diff
@@ -3,7 +3,47 @@
 Sandboxes
 =========
 
-To get a flavor of what Envoy has to offer, we are releasing a
+The docker-compose sandboxes give you different environments to test out Envoy's
+features. As we gauge people's interests we will add more sandboxes demonstrating
+different features.
+
+Locally building a docker image with an envoy binary
```

Comment:
This section should come last, as it should not be required to run the demos. The demo docs should start with a quick start on how to get up and running quickly with already pushed images.